index.js

```diff
@@ -477,6 +477,9 @@ module.exports = function (config, deps) {
         .end(
         function (err, res) {
           if (err != null) {
+            if (err.status !== 201) {
```

Comment:
Perhaps, yes. But after a bit of tracing through all three repositories and trying to figure out the expectations of the errors returned in every use case, I decided that it was "safer" to just try and exactly mimic the current behavior in the case that the status is not a `201` (visible a little lower).